tests/conf.js

```diff
@@ -133,11 +132,11 @@ const login = browser => {
   browser.driver.findElement(by.id('login')).click();
   // Login takes some time, so wait until it's done.
   const bootstrappedCheck = () =>
-    element(by.css('.app-root.bootstrapped')).isPresent();
+    element(by.css('.inbox.bootstrapped')).isPresent();
```

Comment:
The `element(by.css('.app-root.bootstrapped')).isPresent();` works here. I wonder if your change was before the fix was merged. Can you update?